exchanges/bitmex/bitmex_websocket.go

```diff
@@ -508,11 +508,11 @@ func (b *Bitmex) processOrderbook(data []OrderBookL2, action string, p currency.
 					data[i].Side)
```

Comment:
Please disable asset weboskcet for `downsideprofitcontract` and `index`. They don't support orderbooks, but also don't receive updates via websocket